xbmc/utils/EGLImage.cpp

```diff
@@ -109,3 +109,40 @@ void CEGLImage::DestroyImage()
 {
   m_eglDestroyImageKHR(m_display, m_image);
 }
+
+int CEGLImage::GetColorSpace(int colorSpace, int width, int height)
+{
+  switch(colorSpace)
+  {
+    case 10: // BT2020_CL
```

Comment:
magic numbers?

Reply:
I'm not sure what the best thing to do is. I guess I can just bring in a `libavutil/pixfmt.h` include

https://github.com/FFmpeg/FFmpeg/blob/master/libavutil/pixfmt.h#L486